app/src/main/java/com/asfoundation/wallet/billing/share/ShareLinkRepository.kt

```diff
@@ -7,5 +7,5 @@ interface ShareLinkRepository {
   fun getLink(domain: String, skuId: String?, message: String?,
               walletAddress: String,
               originalAmount: String?,
-              originalCurrency: String?): Single<String>
+              originalCurrency: String?, paymentMethod: String): Single<String>
```

Comment:
This could use a code style update.
There are lines with a singe arguments and others with multiple